relay/api/app.py

```diff
@@ -116,6 +117,7 @@ def add_resource(resource, url):
 
     if trustlines.enable_relay_meta_transaction:
         add_resource(RelayMetaTransaction, "/relay-meta-transaction")
+        add_resource(MetaTransactionFees, "/meta-transaction-fees")
```

Comment:
That would be a breaking change, I do not think it is worth it. (The new version of the relay would not work with the old version of the clientlib because the endpoint changed url)

Reply:
True. So it is already to late... I guess it is okay since it will not grow further.